### DISCUSSION TITLE
Is it possible to change section heading colors in existing themes?

### DISCUSSION THREAD

**Discussion by alecmedina-code** [Lissy93/dashy#957] [Help and Support]
I am using the dashy-docs theme and would like to know if its possible to change the color of the section headers rather than having them set randomly by the theme?

I see there are section color settings but this seem to apply to the background color only and not the header color.

![image](https://user-images.githubusercontent.com/64091800/197268671-663e8868-4e39-4802-a2ef-ac1b3b88247e.png)

See the linked image above for example.

**Comment by alecmedina-code** [comment]
I did figure out how to set all header colors together by using custom css with:

```
html[data-theme=dashy-docs] .lbl-toggle {
        background: #636363 !important;
        -webkit-box-shadow: 0 4px #444444 !important;
        box-shadow: 0 4px #444444 !important;
}
```

I did not figure out to change each individually.

**Reply by pikmin-rgb** [reply]
Here is an example to change individually in your config the color of a section : 
```
    {
      "name": "Section",
      "displayData": {
        "color": "#46aab3",
        "sortBy": "default",
        "rows": 1,
        "cols": 1,
        "collapsed": false,
        "hideForGuests": false
      },
      "items": [
        {
          "title": "item",
          "url": "https://example.com",
          "target": "newtab",
          "id": "id"
        }
      ]
    }
```

The "color" line is what you looking for :)
